src/mappers/http-error-mapper.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+
+const { HttpError } = require('@uphold/http-errors');
+const { status } = require('grpc');
```

Comment:
If we're only using `grpc` for error codes, would it make sense to replace them with the actual numeric values? To delete the dependency and avoid it getting included in projects depending on this package.